src/styles/sidebar/components/datalist.scss

```diff
@@ -0,0 +1,41 @@
+@use "../../variables" as var;
+
+.datalist {
+  color: var.$white;
+}
+
+.datalist__items {
+  position: absolute;
+  width: 100%;
+  padding: 0;
+  background-color: var.$grey-mid;
```

Comment:
I guess you are using a dark theme locally and have gone with colors to match that? So this fits in with whatever the user's configuration is locally, I was going to suggest that the default colors assume a light theme and then we use `prefers-color-scheme` media queries in CSS [to adapt to Dark Mode](https://css-tricks.com/dark-modes-with-css/). Does that sound reasonable?

We can do this adaptation on a per-component basis and potentially in future we could also use CSS variables for application-wide color changes.

Reply:
@robertknight @LMS007 I could certainly help provide appropriate CSS for dark-theming—it’s already built into the Pattern Library stuff however

Reply:
Yes, I originally had all that in the Datalist, but moved it out as I needed the keyboard events to work off the input, but I think I can change the props around and move the logic for the carousel navigation into the DL component

Reply:
@jaredpdesigns , yes I think two styles here would be good, a light and dark theme. Pretty much the only css is a bit for the little arrow when the list is open and then the highlight colors for keyboard navigation and a slightly lighter color for mouse hover. But I'm open to anything here.

I also don't have any transitions. IF we added a fade transition, it gets . a bit more complicated with state and re-flow, and I could do that on a second PR.